input/fsh/models.fsh

```diff
@@ -80,9 +83,12 @@ and criteria are defined by FHIRPath expressions.
   * from 0..1 BackboneElement "A convenience to select values relative to some parent FHIRPath."
     * expression 1..1 string "FHIRPath expression for the parent path to select values from"
     * select 0..* contentReference #ViewDefinition.select "See select" "Nested select"
-  * forEach 0..1 BackboneElement "Expression unnest a new row for each item in the specified FHIRPath expression."
+  * forEach 0..1 BackboneElement "Unnests a new row for each item in the specified FHIRPath expression."
+    * expression 1..1 string "FHIRPath expression for the parent path to select values from"
+    * select 0..* contentReference #ViewDefinition.select "See select" "Nested select"
+  * forEachOrNull 0..1 BackboneElement "Unnests a new row for each item in the specified FHIRPath expression, generating a null value if empty"
```

Comment:
```suggestion
  * forEachOrNull 0..1 BackboneElement "Unnests a new row for each item in the specified FHIRPath expression, generating a null value if empty"
    * expression 1..1 string "FHIRPath expression for the parent path to select values from"
    * select 0..* contentReference #ViewDefinition.select "See select" "Nested select"
```

I think this should match the structure for `forEach`?

Reply:
Shall I add an invariant rule to only allow `forEach` or `forEachOrNull` but not both?

Reply:
If you can only have one or the other, let's not have parallel things here -- e.g. instead of both`forEach` and `forEachOrNull` and add an option that defaults to false in one `forEach` -- e.g. `forEach.orNull = true`